src/event.rs

```diff
@@ -317,6 +317,25 @@ pub enum WindowEvent {
         ///
         /// This value may be NaN.
         delta: f64,
+        /// Only available on **iOS**.
+        /// https://developer.apple.com/documentation/uikit/uipinchgesturerecognizer/1622233-velocity?language=objc
+        velocity: Option<f32>,
+        phase: TouchPhase,
+    },
+
+    /// Two-finger pan gesture
+    ///
+    /// ## Platform-specific
+    ///
+    /// - Only available on **iOS**.
+    /// - On iOS, not recognized by default. It must be enabled when needed.
+    PanGesture {
+        device_id: DeviceId,
+        /// Change in pixels of pan gesture from last update.
+        delta: PhysicalPosition<f32>,
+        /// Velocity of pan (delta/time?). Only available on **iOS**
+        /// https://developer.apple.com/documentation/uikit/uipangesturerecognizer/1621209-velocityinview?language=objc
+        velocity: PhysicalPosition<f32>,
```

Comment:
As a developer using Winit, I'd still have no idea how to use this value. Maybe an example is in order? Or maybe you could update the example to use it in some (meaningful) way?

Reply:
I believe it's used to trigger things like "inertial" scrolling. On most Apple UIs a fast pan will cause the document to continue scrolling with decaying velocity after the pan event is over.

I think it could be useful to game developers, for example a low velocity pan could be used to make a character walk, but a high velocity pan could make them jump. I don't particularly have an immediate use for it and I don't really want to develop a toy example. :)

Here's an example that implements dynamic scrolling by manually calculating the scroll velocity:
https://macresearch.org/dynamics-scrolling/

This API gives it for free, so I figured it saves some developers some effort.

Reply:
I removed velocity from the events to simplify the PR. @madsmtm